materials/validators.py

```diff
@@ -1,10 +1,9 @@
 from rest_framework import serializers
+from urllib.parse import urlparse
 
-class URLValidator:
-    def __call__(self, value):
+def URLValidator(value):
```

Comment:
Названия функций нужно писать в snake_case
```suggestion
def url_validator(value):
```